src/modules/Lan.rb

```diff
@@ -975,50 +969,24 @@ def ProposeVirtualized
       nil
     end
 
+    # Proposes additional packages when needed by current networking setup
+    #
     # @return [Array] of packages needed when writing the config
     def Packages
-      # various device types require some special packages ...
-      type_requires = {
-        # for wlan require iw instead of wireless-tools (bnc#539669)
-        "wlan" => "iw",
-        "vlan" => "vlan",
-        "tun"  => "tunctl",
```

Comment:
The changelog and the PR title only mention vlan.

Reply:
both updated